src/v3/src/components/ConsentHeader/ConsentHeader.tsx

```diff
@@ -69,12 +68,11 @@ const ConsentHeader: FunctionComponent = () => {
       >
         {typeof href !== 'undefined' && typeof logoHref !== 'undefined'
           ? (
-            <Box component="div">
+            <Box>
               <Link
                 href={href}
                 target="_blank"
                 rel="noopener noreferrer"
-                aria-label={altText}
```

Comment:
aria-label is probably required here since this is an anchor tag with no text.

Reply:
Got it. I think I saw this issue raised as an `a11y` error at some point but this PR was also giving me weird test failures so I reverted to get it back to this state. I think the `a11y` runs via e2e tests that I didn't run locally though so I'm guessing Bacon will tell me you're right. This means we'll have to import this `Link` component from `@mui/material` since Odyssey `Link` doesn't support aria-label, but that's not the end of the world

Reply:
Oh interesting, did Odyssey say they will eventually support that prop? That seems like a standard thing to allow especially given how anchors can wrap anything (and not necessarily rely on the inner content to represent the text for screen readers).

Reply:
I think they expect that `Link` components will always render some sort of text, though I can ask